SpectralCubeReprojectMaskExample_v3.ipynb

```diff
@@ -0,0 +1,1546 @@
+      "text/plain": [
+       "<Figure size 432x288 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "masked_moment1 = masked_cube.moment1()\n",
+    "\n",
+    "ax = plt.subplot(projection=masked_moment1.wcs)\n",
+    "im = ax.imshow(masked_moment1.value, origin='lower', cmap='coolwarm')\n",
+    "cbar = plt.colorbar(im)\n",
+    "cbar.set_label('Centroid (km/s)')\n",
+    "\n",
+    "ax.set_ylabel('Declination')\n",
+    "ax.set_xlabel('Right Ascension')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The color bar on the moment 1 map is skewed due to the presence of at least one outlier pixel with a large negative velocity."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Step 9. Masking Outliers\n",
```

Comment:
@abulatek -- does increasing `high_min_pixels` or not using the binary dilation step help with removing the outliers in the last step?

Reply:
First test: increasing `high_min_pixels` from 10 to 20. Using the colorbar in the moment 1 map as a diagnostic for whether there are fewer outliers. There are still outliers even when increasing `high_min_pixels`. high_min_pixels = 10 (left) versus high_min_pixels = 20 (right). 

<img width="620" alt="high_min_pixels_test" src="https://user-images.githubusercontent.com/28034735/166565574-c185a314-f262-489a-9d9f-de63c0400daf.png">

Second test: `high_min_pixels` back to 10; not using binary dilation. This does seem to help with outliers somewhat! Binary dilation (left) versus no binary dilation (right). Range of velocities is a little more reasonable, and looking at number of outliers (plots below, same left/right definitions) shows that they are fewer. I'll separate binary dilation into a separate sub-step, but keep it turned off for now.

<img width="608" alt="binary_dilation_test" src="https://user-images.githubusercontent.com/28034735/166565592-c0bae0c0-2b84-4511-8315-85f4bbc2afb4.png">

<img width="641" alt="binary_dilation_test_outliers" src="https://user-images.githubusercontent.com/28034735/166565665-814f3a81-c24f-45e5-a3e0-e2ebade8356e.png">